build/subgraph.yaml

```diff
@@ -3,12 +3,41 @@ schema:
   file: schema.graphql
 dataSources:
   - kind: ethereum
-    name: AgentFactory
+    name: AgentFactoryOld
     network: gnosis
     source:
       address: "0x88DE734655184a09B70700aE4F72364d1ad23728"
       abi: AgentFactory
       startBlock: 27911512
+    mapping:
+      kind: ethereum/events
+      apiVersion: 0.0.7
+      language: wasm/assemblyscript
+      entities:
+        - CreateMech
+        - OwnerUpdated
+        - Pause
+        - Unpause
+      abis:
+        - name: AgentFactory
+          file: AgentFactoryOld/abis/AgentFactory.json
```

Comment:
Are you sure the paths are right? because it's the new factory but the old path?